mits/site_sections/mits_admin.py

```diff
@@ -9,8 +9,96 @@ def index(self, session, message=''):
             'teams': session.mits_teams()
         }
 
+    def create_new_application(self):
+        cherrypy.session.pop('mits_team_id', None)
+        raise HTTPRedirect('../mits_applications/team')
```

Comment:
This fixes a minor bug in which clicking the "Create a New Application" button on the admin side would just take you the already-logged-in application if you'd previously logged in as one of the teams.